src/api/PluginManager.ts

```diff
@@ -43,7 +53,10 @@ export async function startPlugins() {
                 zip.closeEntry();
 
                 if (manifest.name in plugins) throw new Error(`Plugin ${manifest.name} already registered`);
-                if (!isPluginEnabled(manifest.name)) continue;
+                if (!isPluginEnabled(manifest.name)) {
+                    disabledPlugins[manifest.name] = manifest;
+                    continue;
+                }
 
                 zip.openEntry("index.js.bundle");
                 const pluginBuffer = zip.readEntry("binary");
```

Comment:
can you make this use the loadPluginBundle func as well please

Reply:
Will do when I’m home